### ISSUE TITLE
tests/crypto/ecc_dh fails on qemu_nios2

### ISSUE THREAD

**Issue by nashif** [zephyriot/zep-jira14#2291]
**_Reported by Anas Nashif:_**

After updating Tinycrypt to the latest, we have this test failing in Qemu.

```
                          ***** BOOTING ZEPHYR OS v1.8.99 - BUILD: Aug  3 2017 08:59:24 *****
tc_start() - Performing ECC-DH tests:
Performing cavp_ecdh test:
Test <span>#</span>1: ECDHNIST-p256
  Vector <span>#</span>00 check Z - success
@ /home/buildslave/src/github.com/zephyrproject-rtos/ci-test/zephyr/tests/crypto/ecc_dh/src/../../ecc_dsa/src/test_ecc_utils.c:147:
***** Kernel Panic! *****
Current thread ID: 0x004003ac
Faulting instruction: 0xdeadbaa9
  r1: 0xdeadbaad  r2: 0xdeadbaad  r3: 0xdeadbaad  r4: 0xdeadbaad
  r5: 0xdeadbaad  r6: 0xdeadbaad  r7: 0xdeadbaad  r8: 0xdeadbaad
  r9: 0xdeadbaad r10: 0xdeadbaad r11: 0xdeadbaad r12: 0xdeadbaad
 r13: 0xdeadbaad r14: 0xdeadbaad r15: 0xdeadbaad  ra: 0xdeadbaad
estatus: deadbaad
```

(Imported from Jira ZEP-2460)

**Comment by nashif** [comment]
**_by gurucharan p:_**

I could still observe the issue in Commit ID: c7634a94. i.e on v1.9.0-rc1.
So Could you please let me know in which commit ID this issue has been fixed.

**Comment by nashif** [comment]
**_by Leandro Pereira:_**

The fix was pushed to master after v1.9.0-rc1 was tagged.  Can you please test with dd740c1, or any newer commit?

**Comment by nashif** [comment]
**_by Leandro Pereira:_**

The fix is now part of 1.9.0-rc2.

**Comment by nashif** [comment]
**_by Chandrakala Kempanna:_**

_gurucharan p_ Please Verify

**Comment by nashif** [comment]
**_by gurucharan p:_**

* [J2460.txt](https://s3.amazonaws.com/logs.zephyrproject.org/jira/GH-2291/J2460.txt) (size: 3953)

**Comment by nashif** [comment]
**_by gurucharan p:_**

According to the logs attached  [^J2460.txt] It seems the issue has been fixed in v1.9.0-rc2. So closing the Jira.
